docs/tutorials/10_effective_dimension.ipynb

```diff
@@ -0,0 +1,770 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Effective Dimension of Qiskit Neural Networks\n",
+    "In this tutorial, we will take advantage of the `EffectiveDimension` and `LocalEffectiveDimension` classes to evaluate the power of Quantum Neural Network models. These are metrics based on information geometry that connect to notions such as trainability, expressibility or ability to generalize.\n",
```

Comment:
Here we introduce these two classes, one is simply EffectiveDimension and the other LocalEffectiveDimension. In subsequent text is has "**global** effective dimension" and "**local** effective dimension" with the aspects highlighted. Now the local one seems to match to the LocalEffectiveDimension and I guess we leave it to readers to infer that the global one is simply the EffectiveDimension. Maybe we could explicitly make the connection in the text between the concept i.e; global effective dimension and the class implementing it i.e. EffectiveDimension.